src/utils/auth-helper.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Fetches config from environment and API.
+ * @returns A promise to get & set session storage keys with appropriate values.
+ */
+/** Gets Keycloak JWT and parses it. */
```

Comment:
This comment goes with line 6.

Reply:
oops, sloppy copy pasta here! Fixed in next commit